### PR TITLE
[Phpspec] Add missing specs on customer core model

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Branch?         | 1.3
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Comment by loic425** [comment]
@pamil Could you restart 3.4 build please ?

**Comment by lchrusciel** [comment]
done

**Comment by loic425** [comment]
> done

Thanks Łukasz, wait for it... success 💪

File: src/Sylius/Component/Core/spec/Model/CustomerSpec.php [hunk 1]
```diff
@@ -64,4 +66,39 @@ function it_adds_address_when_billing_address_is_set(AddressInterface $address):
...
+    function it_resets_customer_of_previous_user(ShopUserInterface $previousUser, ShopUserInterface $user)
+    {
+        $this->setUser($previousUser);
+
+        $previousUser->setCustomer(null)->shouldBeCalled();
+
+        $this->setUser($user);
+    }
+
+    function it_does_not_replace_user_if_it_is_already_set(ShopUserInterface $user)
```
Comment (Zales0123) [inline src/Sylius/Component/Core/spec/Model/CustomerSpec.php]: Missing return `: void` for this and some other functions :)

Comment (loic425) [reply src/Sylius/Component/Core/spec/Model/CustomerSpec.php]: > Missing return `: void` for this and some other functions :)

Fixed! thanks

**Comment by lchrusciel** [comment]
Thanks, Loïc! :1st_place_medal:

**Pull request merged**